mysterious-mice/spacebook/game/utils.py

```diff
@@ -122,74 +144,134 @@ def command_move(game_data, direction):
     old_position = game_data["rover"]
     success = True
 
+    # check if current position contains an obstacle
+    for obstacle in game_data["obstacles"]:
+        if (
+            game_data["obstacles"][obstacle][0] == old_position[0]
+            and game_data["obstacles"][obstacle][1] == old_position[1]
+        ):
+            # if the obstacle is a dust stom there is a 50% chace power comsumption will increase
+            if obstacle == "dust_storm":
+                if random.randint(1, 10) > 5:
+                    game_data["power_usage"] += 5
+                    game_data["messages"].append(
+                        {
+                            "from_rover": False,
+                            "message": f"Sand has gotten stuck in the treads. Moving becomes less efficient.",
```

Comment:
Doesn't need to be an f-string since no variables are passed, the linter will call this out